elide-core/src/main/java/com/yahoo/elide/core/EntityDictionary.java

```diff
@@ -187,8 +186,8 @@ public ParseTree getPermissionsForClass(Class<?> resourceClass,
      *         or {@code null} if the permission is not specified on that field
      */
     public ParseTree getPermissionsForField(Class<?> resourceClass,
-                                                       String field,
-                                                       Class<? extends Annotation> annotationClass) {
+            String field,
```

Comment:
We really need a style guide -.- There are probably 3 "acceptable" styles for multi-line method signatures:

Namely:

  * Something that resembles the original
  * Something that resembles the new version
  * One that lines up the _parameters_:
      * i.e. ```
public ParseTree getPermissionsForField(Class<?> resourceClass,
                                                                      String field,
                                                                      Class<? extends Annotation> annotationClass) {
         ```

tbh, I am not particular on any such style so long as we're consistent.